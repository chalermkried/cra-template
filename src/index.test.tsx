// eslint-disable-next-line @typescript-eslint/no-unused-vars
import React from 'react';

describe('index', () => {
  it('should match snapshot', () => {
    const rootEl = document.createElement('div');
    rootEl.id = 'root';
    document.body.append(rootEl);

    // eslint-disable-next-line @typescript-eslint/no-floating-promises
    (() => import('./index'))();

    expect(document.body).toMatchSnapshot();
  });
});
