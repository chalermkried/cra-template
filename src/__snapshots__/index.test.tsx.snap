// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index should match snapshot 1`] = `
<body>
  <div
    id="root"
  />
</body>
`;
